txscript/engine.go

```diff
@@ -198,6 +205,8 @@ func (vm *Engine) curPC() (script int, off int, err error) {
 	return vm.scriptIdx, vm.scriptOff, nil
 }
 
+//func (vm *Engine) verifyWitnessProgram(uint version, wi
```

Comment:
Yeah this was left over as an un-finished thought. I've extracted out the new code in `Step` to a separate function.